### DISCUSSION TITLE
sea-orm-migrations: implement command for deleting database

### DISCUSSION THREAD

**Discussion by tp971** [SeaQL/sea-orm#1460] [Ideas]
Right now, there is no general way to delete all tables in a database and not run any migrations. The command `fresh` re-runs all migrations, and the command `reset` seems to do this, but it only works if all migrations can be undone (implement `down`). Therefore, I suggest a command like `clean` that just deletes all tables in the database and does nothing else.

**Comment by tyt2y3** [comment]
Sounds like an interesting idea
